scene/resources/theme.cpp

```diff
@@ -1613,6 +1613,11 @@ void Theme::merge_with(const Ref<Theme> &p_other) {
 		}
 	}
 
+	// Defaults.
+	set_default_font(p_other->default_font);
```

Comment:
This should probably be optional, at minimum it should only assign if the other has these set, so:
```suggestion
	if (p_other->has_default_font()) {
		set_default_font(p_other->default_font);
	}
```
And similar for the rest

Reply:
Yes you are right, thanks !